### PR TITLE
Make lowercase settings fatal

### PR DESCRIPTION
**Pull request by foursixnine**
Avoid red herring, by removing a warning that should be a fatal error.

https://progress.opensuse.org/issues/113528#note-9

### CODE REVIEW

**Comment by okurz** [comment]
I am thinking, should we bump the testing
test API version?

**Comment by foursixnine** [comment]
> I am thinking, should we bump the testing test API version?

makes sense, done

**Review by kalikiana — APPROVED** [review]

**Review by okurz — APPROVED** [review]
change is fine but for new behaviour enforcing stricter settings I would like to give more time for others to see this PR to be ready for that. Thinking of roughly 2 weeks before we merge this.

**Review by mimi1vx — APPROVED** [review]

**Comment by grisu48** [comment]
Please let's not merge this in August when everyone is on vacation. A change like this might stress out the remaining skeleton crew more then it should.

I vote for postponing this until September when the vacation time is over.

**Review by kraih — APPROVED** [review]

**Review by grisu48 — APPROVED** [review]

**Comment by kalikiana** [comment]
So it seems @grisu48 is in favor of (updating and) merging this change now. Note that the *not-ready* still keeps this from being merged. I would suggest to remove that **Monday September 5** if others are fine with it (since that's the start of the week).

**Review by punkioudi — APPROVED** [review]

**Comment by kalikiana** [comment]
@mergifyio rebase

**Comment by AdamWill** [comment]
> Shouldn't be (I'm downplaying it :p), I would not expect users to rely on lower case settings.

Well, this user was...in our tests we do [stuff like this](https://pagure.io/fedora-qa/os-autoinst-distri-fedora/blob/main/f/lib/utils.pm#_718) as an internal state marker:

    set_var("_welcome_done", 1);

as it's entirely internal to the test logic I didn't bother hitting caps lock for those variables. :/ Guess I'll go update them all to upper case now.

**Pull request merged**
